config.d.ts

```diff
@@ -0,0 +1,22 @@
+export interface Config {
```

Comment:
could we move this to a `types` directory or something?

Reply:
Moved it to `packages/app/src/config`